lib/haml_lint/linter/rubocop.rb

```diff
@@ -71,6 +71,7 @@ def extract_lints_from_offenses(offenses, source_map)
     # @return [Array<String>]
     def rubocop_flags
       flags = %w[--format HamlLint::OffenseCollector]
+      flags += %w[--except Style/EndOfLine]
```

Comment:
We have a canonical list of disabled cops in the [default configuration](https://github.com/brigade/haml-lint/blob/4e7f18381f1151d9a628fcdbf632c69ca72dc72b/config/default.yml#L63-L80). The `Style/EndOfLine` cop should be added there instead. Thanks!